README.md

```diff
@@ -40,8 +40,6 @@ Bookmarks
 
 [Offensive Security 2013 - FSU - Lecture5: Reverse Engineering (x86) Workshop Day 1 - YouTube](https://www.youtube.com/watch?v=Pg8bmV9vcKg&feature=youtu.be)
 
-[Offensive Computer Security Home Page Spring 2014](http://www.cs.fsu.edu/~redwood/OffensiveComputerSecurity/)
```

Comment:
This could be replaced with the lecture playlist on youtube: https://www.youtube.com/playlist?list=PLl2ZIjyGEWWVYUeYLTkanteQo4BPephik